src/components/PlacementResult.js

```diff
@@ -30,16 +33,41 @@ export default class PlacementResult extends React.Component<PlacementResultDefa
     linkText: null,
     imageUrl: null,
     markup: null,
+    markupCount: -1,
   }
 
   static displayName = 'PlacementResult';
 
+  renderMarkupIframe() {
+    const getBlobURL = (code) => {
+      const blob = new Blob([code], { type: 'text/html' });
+      return URL.createObjectURL(blob);
+    };
+
+    const source = `
+      <html>
+        <body>
+          ${this.props.markup || ''}
+        </body>
+      </html>
+    `;
+    const iframeSource = getBlobURL(source);
+
+    return (
+      <iframe
+        title={`Markup Iframe ${this.props.markupCount}`}
```

Comment:
We can probably change this to something a little more user friendly like "Promotion"... There's no reason it needs to be unique in the rare case there are multiple markup tags placements...